README.md

```diff
@@ -4,39 +4,62 @@
 [![Build Status](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts.svg?style=shield)](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts)
 [![Coverage Status](https://codecov.io/gh/OpenZeppelin/openzeppelin-contracts/graph/badge.svg)](https://codecov.io/gh/OpenZeppelin/openzeppelin-contracts)
 
-**OpenZeppelin Contracts is a library for secure smart contract development.** It provides implementations of standards like ERC20 and ERC721 which you can deploy as-is or extend to suit your needs, as well as Solidity components to build custom contracts and more complex decentralized systems.
+*A library for secure smart contract development.* Build on a solid foundation of community-vetted code.
```

Comment:
Bold is much better IMO. Please remember to use bold! I know it's that your editor shows it differently.
```suggestion
**A library for secure smart contract development.** Build on a solid foundation of community-vetted code.
```

Reply:
It's not my editor, it's me not paying attention :man_facepalming:

Thanks!